docs/content/development.md

```diff
@@ -14,7 +14,7 @@ The main backend source code is in the [dcmgr](https://github.com/axsh/wakame-vd
 
 The file [dcmgr/lib/dcmgr.rb](https://github.com/axsh/wakame-vdc/blob/master/dcmgr/lib/dcmgr.rb) provides a nice overview of the code. It shows the module and class structure.
 
-#### WebAPI
+### WebAPI
 
 The WebAPI code uses [Sinatra](http://www.sinatrarb.com) and is in the [dcmgr/lib/dcmgr/endpoints/12.03](https://github.com/axsh/wakame-vdc/tree/master/dcmgr/lib/dcmgr/endpoints/12.03) directory.
 
```

Comment:
A change from h4 to h3 is indeed very subtle in this theme but we should enforce a standard anyway. It's cleaner and will make our lifes easier if we eventually switch to another theme where the difference might not to be so subtle.